shared-libs/transitions/src/transitions/update_scheduled_reports.js

```diff
@@ -19,7 +20,7 @@ const getLeafPlaceTypeIds = () => {
 const getParentId = contact => {
   const parentType = contact &&
                      contact.parent &&
-                     (contact.parent.contact_type || contact.parent.type);
+                     contactTypeUtils.getTypeId(contact.parent);
```

Comment:
Minor: No need to check the parent twice... 

```
const parentType = contact && contactTypeUtils.getTypeId(contact.parent);
```

Reply:
Thanks! Done :)